react/components/PriceRange.js

```diff
@@ -1,80 +1,104 @@
-import React, { useRef } from 'react'
-import PropTypes from 'prop-types'
-import { useRuntime } from 'vtex.render-runtime'
-import { useIntl } from 'react-intl'
-import { Slider } from 'vtex.styleguide'
-import { formatCurrency } from 'vtex.format-currency'
-
-import { facetOptionShape } from '../constants/propTypes'
-import { getFilterTitle } from '../constants/SearchHelpers'
-import FilterOptionTemplate from './FilterOptionTemplate'
-import useSearchState from '../hooks/useSearchState'
-
-const DEBOUNCE_TIME = 500 // ms
+import React, { useRef } from "react";
+import PropTypes from "prop-types";
+import { useRuntime } from "vtex.render-runtime";
+import { useIntl } from "react-intl";
+import { Slider } from "vtex.styleguide";
+import { formatCurrency } from "vtex.format-currency";
+
+import { facetOptionShape } from "../constants/propTypes";
+import { getFilterTitle } from "../constants/SearchHelpers";
+import FilterOptionTemplate from "./FilterOptionTemplate";
+import useSearchState from "../hooks/useSearchState";
+import { usePixel } from "vtex.pixel-manager";
+import { useSearchPage } from "vtex.search-page-context/SearchPageContext";
+
+const DEBOUNCE_TIME = 500; // ms
 
 /** Price range slider component */
 const PriceRange = ({ title, facets, priceRange }) => {
-  const { culture, setQuery } = useRuntime()
-  const intl = useIntl()
+  const { culture, setQuery } = useRuntime();
+  const intl = useIntl();
+
+  const { push } = usePixel();
+  const { searchQuery } = useSearchPage();
+
+  const navigateTimeoutId = useRef();
 
-  const navigateTimeoutId = useRef()
+  const { fuzzy, operator, searchState } = useSearchState();
 
-  const { fuzzy, operator, searchState } = useSearchState()
+  const getCategoryFromObjs = (products) => {
+    const categoryId = products[0].categoryId;
+    const result = products.filter((obj) => obj.categoryId !== categoryId);
+    return result.length == 0 ? categoryId : "";
+  };
```

Comment:
Same here

Reply:
Done in last commit (required operations)